sympy/geometry/tests/test_parabola.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import division
+
+from sympy import oo
+from sympy.geometry import Line, Point, Point2D, Parabola
```

Comment:
@asmeurer  @sampadsaha5 Thank you for the feedback. We are working to correct them!